tools/interop_matrix/client_matrix.py

```diff
@@ -145,6 +145,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
         ('v1.18.0', ReleaseInfo(runtimes=['java_oracle8'])),
         ('v1.19.0', ReleaseInfo(runtimes=['java_oracle8'])),
         ('v1.20.0', ReleaseInfo(runtimes=['java_oracle8'])),
+        ('v1.21.0', ReleaseInfo(runtimes=['java_oracle8']))),
```

Comment:
syntax error?
https://source.cloud.google.com/results/invocations/c64b5708-4eb0-4464-8993-c82e825e1420/targets/github%2Fgrpc%2Fsanity_linux_dbg_native/tests

Reply:
arg. Yeah. Thanks for fixing it.